addon/services/log.js

```diff
@@ -314,6 +318,32 @@ export default Ember.Service.extend(Ember.Evented, {
   */
   showPromiseErrors: false,
 
+  /**
+    Flag: indicates whether log service will skip error messages that defined in errorMessageFilters array variable.
+
+    @property errorMessageFilterActive
+    @type Boolean
+    @default false
+    @example
+    ```
+    // Log service 'errorMessageFilterActive' setting could be also defined through application config/environment.js
+    module.exports = function(environment) {
+      var ENV = {
+        ...
+        APP: {
+          ...
+          log: {
+            enabled: true,
+            errorMessageFilterActive: true
+          }
+          ...
+        }
+        ...
+    };
+    ```
+  */
+  errorMessageFilterActive: false,
```

Comment:
The comment on this property gives the false impression that in the `errorMessageFilters` array I can describe which error messages should be excluded.
I think you need to rename this option to something similar to `skipTransitionAbortedErrors`. Or give the ability to add exclusions to the `errorMessageFilters` array.
I am inclined to rename, because I do not see any cases of using this feature, except for the `TransitionAborted` errors.